src/lib/runtime/sandbox/Web3FunctionDockerSandbox.ts

```diff
@@ -49,7 +49,6 @@ export class Web3FunctionDockerSandbox extends Web3FunctionAbstractSandbox {
     args.push("run");
     args.push(`--allow-env=WEB3_FUNCTION_SERVER_PORT`);
     args.push(`--allow-net`);
-    args.push(`--unstable`);
```

Comment:
I think we rely on this flat, because we use an unstable deno api in the web3-functions-sdk to expose the small webserver to receive the start message with context data.

This is the API we use, which is unstable:
https://deno.land/api@v1.31.1?unstable=&s=Deno.serve

We should instead use the stable one:
https://deno.land/api@v1.31.1?unstable=&s=Deno.serveHttp
which is a bit more complex, and the reason why I didn't use it in the first place.

Note that removing the `--unstable` flag will be a braking change, as it will breack executions of all previous web3 functions.

Let's handle it in another PR